### PR TITLE
Use ocamlformat

### PR DESCRIPTION
**Pull request by smorimoto**
It's a good idea to use ocamlformat to get the consistent code layout. ocaml-lsp works very well with it and gives you a good experience.

### CODE REVIEW

**Comment by smorimoto** [comment]
@ksuenaga Ready to merge!

File: miniml-compiler/lib/closure.ml [hunk 1]
```diff
@@ -11,102 +12,104 @@ type binOp = S.binOp
...
-        (text "(" <*>
-         (List.fold_left
-            (fun d v -> d <*> text "," <+> pr_of_value v)
-            (pr_of_value v) vs')
-         <*> (text ")"))
+        text "("
+        <*> List.fold_left
+              (fun d v -> d <*> text "," <+> pr_of_value v)
+              (pr_of_value v) vs'
+        <*> text ")"
```
Comment (smorimoto) [inline miniml-compiler/lib/closure.ml:54]: Much better

File: miniml-compiler/lib/closure.ml [hunk 2]
```diff
@@ -11,102 +12,104 @@ type binOp = S.binOp
...
+          (nest 2
+             (group
+                (text "let"
+                <+> text "rec"
+                <+> text id
+                <+> pr_of_ids parms
+                <+> text "="
+                <|> pr_of_exp 1 body))
+          <+> text "in"
+          <|> pr_of_exp 1 e)
```
Comment (smorimoto) [inline miniml-compiler/lib/closure.ml:103]: Readable

File: miniml-compiler/lib/flat.ml [hunk 3]
```diff
@@ -3,116 +3,128 @@ module S = Syntax
...
-                    nest 2 (text "=" <|> pr_of_exp 1 body)))))
+        group
+          (text "let"
+          <+> text "rec"
+          <+> group
+                (text id
+                <+> text "("
+                <*> tparms
+                <*> text ")"
+                <+> nest 2 (text "=" <|> pr_of_exp 1 body)))
```
Comment (smorimoto) [inline miniml-compiler/lib/flat.ml:119]: Readable

File: miniml-compiler/lib/normal.ml [hunk 4]
```diff
@@ -10,94 +11,97 @@ type binOp = S.binOp
...
+          (nest 2
+             (group
+                (text "let"
+                <+> text "rec"
+                <+> text id
+                <+> text v
+                <+> text "="
+                <|> pr_of_exp 1 body))
+          <+> text "in"
+          <|> pr_of_exp 1 e)
```
Comment (smorimoto) [inline miniml-compiler/lib/normal.ml:87]: Readable

File: miniml-compiler/lib/reg.ml [hunk 5]
```diff
@@ -1,108 +1,126 @@
...
+      ^ "call"
+      ^ tab
+      ^ string_of_dest dst
+      ^ ", "
+      ^ string_of_operand tgt
+      ^ "("
+      ^ string_of_operand a0
+      ^ ", "
+      ^ string_of_operand a1
+      ^ ")"
```
Comment (smorimoto) [inline miniml-compiler/lib/reg.ml:86]: Readable

File: miniml-compiler/lib/reg.ml [hunk 6]
```diff
@@ -1,108 +1,126 @@
...
-  "proc " ^ lbl ^ "(" ^ string_of_int n ^ ") =\n" ^
-  (String.concat "\n"
-     (List.map (fun i -> string_of_instr "  " "\t" i) instrs)) ^ "\n"
+  "proc "
+  ^ lbl
+  ^ "("
+  ^ string_of_int n
+  ^ ") =\n"
+  ^ String.concat "\n" (List.map (fun i -> string_of_instr "  " "\t" i) instrs)
+  ^ "\n"
```
Comment (smorimoto) [inline miniml-compiler/lib/reg.ml:116]: Readable

File: miniml-compiler/bin/main.ml [hunk 7]
```diff
@@ -1,78 +1,72 @@
 let debug = ref false
-let dprint s = if !debug then (print_string (s ()) ; flush stdout)
+
+let dprint s =
+  if !debug then (
+    print_string (s ());
+    flush stdout)
```
Comment (smorimoto) [inline miniml-compiler/bin/main.ml:6]: Readable

File: miniml-compiler/lib/cfg.ml [hunk 8]
```diff
@@ -55,42 +59,51 @@ let rec flatmap_of_string f s =
...
-let dot_of_cfg lbl str_of_prop cfg  =
+let dot_of_cfg lbl str_of_prop cfg =
   let escape_char = function
-      '<' -> "\\<" | '>' -> "\\>" | '{' -> "\\{" | '}' -> "\\}"
-    | c -> String.make 1 c in
+    | '<' -> "\\<"
+    | '>' -> "\\>"
+    | '{' -> "\\{"
+    | '}' -> "\\}"
+    | c -> String.make 1 c
```
Comment (smorimoto) [inline miniml-compiler/lib/cfg.ml:68]: Consistent

**Review by ksuenaga — APPROVED** [review]
Looks good!

**Comment by ksuenaga** [comment]
Thank you very much for your contribution!

**Pull request merged**